palindrom-client.html

```diff
@@ -186,7 +186,13 @@
                     listenTo = typeof listenTo == "string" ? document.getElementById(listenTo) : listenTo;
                 }
                 let palindrom = PalindromDOM.instances && PalindromDOM.instances.get(this.remoteUrl);
-                if(!palindrom){
+                if(!palindrom) {
+                    /**
+                     * To tell whether we connected to an existing instance,
+                     * or created our own. This is useful for detachement, we don't want
+                     * to destroy Palindrom instances that we don't own 
+                     * */
+                    this.isUsingOwnInstanceOfPalindrom = true;
```

Comment:
:+1:
If we are adding it to the public API, I'd document it.

However, in long term, I think `palindrom-client` could not create and instance at all.